tricat_211/CircleObstacle[] circles
